src/test/scala/io/iohk/ethereum/forkid/ForkIdSpec.scala

```diff
@@ -51,9 +51,11 @@ class ForkIdSpec extends AnyWordSpec with Matchers {
         Some(9069000)
       ) // First and last Constantinople, first Petersburg block
       create(9068999) shouldBe ForkId(0x668db0afL, Some(9069000)) // Last Petersburg block
-      // TODO: Add Muir Glacier and Berlin
-      create(9069000) shouldBe ForkId(0x879d6e30L, None) // First Istanbul block
-      create(12644529) shouldBe ForkId(0x879d6e30L, None) // Today Istanbul block
+      create(9069000) shouldBe ForkId(0x879d6e30L, Some(9200000)) // First Istanbul block
+      create(9200000 - 1) shouldBe ForkId(0x879d6e30L, Some(9200000)) // Last Istanbul block
+      create(9200000) shouldBe ForkId(0xe029e991L, None) // First Muir Glacier block
```

Comment:
Don't we miss Berlin somewhere after Muir Glacier ?

Reply:
We do, it's going to be added after Muir is in